SYCL/lit.cfg.py

```diff
@@ -336,7 +314,7 @@
 if 'cpu' in config.target_devices.split(','):
     found_at_least_one_device = True
     lit_config.note("Test CPU device")
-    cpu_run_substitute = "env SYCL_DEVICE_FILTER={SYCL_PLUGIN}:cpu,host ".format(SYCL_PLUGIN=config.sycl_be)
+    cpu_run_substitute = "env SYCL_DEVICE_FILTER={SYCL_PLUGIN}:cpu ".format(SYCL_PLUGIN=config.sycl_be)
```

Comment:
We should outline those small `,host` changes into a separate PR: I'm concerned that in some cases tests might be launched on host while it is still there. In combination with removal of code pieces like:
```c++
if (Queue.get_device().is_host()) {
    std::cout << "Skipping test\n";
    return 0;
  }
```

That could introduce new failures while we removing tests

Reply:
I suggest we move the tests that depend on this to be part of this PR to avoid further inter-dependencies.

Reply:
> I suggest we move the tests that depend on this to be part of this PR to avoid further inter-dependencies.

It is hard to grep for such. Actually, I think that the situation I described won't occur in our CI, but it is theoretically possible

Reply:
Most cases should work as long as the host device isn't the only device. Even if that happens, it should only happen between those tests being merged and this being merged.